lists/ru.csv

```diff
@@ -72,12 +72,14 @@ http://looo.ch,CULTR,Culture,2014-04-15,citizenlab,
 http://lubov-kontakt.ru,DATE,Online Dating,2014-04-15,citizenlab,
 http://lurkmore.to,POLR,Political Criticism,2014-04-15,citizenlab,
 http://milli-mejlis.blogspot.com,HUMR,Human Rights Issues,2014-04-15,citizenlab,
+http://muslimiman.info/articles/veroybejdenie/kitab_tauhid/kitab_tauhid_1/, REL, Religion, 2016-04-29, citizenlab,

```

Comment:
Also the source column should not say citizenlab, but should mention the actual source that added this URL to the list.